SU-SWS/stanford_profile
# D8CORE-938: Menu tweaks.

# READY FOR REVIEW 

# Summary
- Update Webpack builds
- Hide secondary nav at everything less than and equal to the `md` breakpoint on stanford page
- Set branches for dependant work to D8CORE-938
- Update Decanter to 6.0.1

# Review By (Date)
- End of the week

# Urgency
- Medium

# Steps to Test

1. Review code
2. Use the tugboat build at: https://github.com/SU-SWS/acsf-cardinalsites/pull/46 or https://navigation-test.sites.stanford.edu/ to review the hide/show functionality of the navigation split.
3. Test the webpack compile by running `npm run publish` to ensure no further changes.

# Affected Projects or Products
- D8CORE

# Associated Issues and/or People
- D8CORE-901 
- D8CORE-938
- @katrialesser FYI
- @pookmish FYI

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
